packages/components/src/input/src/components/osds-input/osds-input.tsx

```diff
@@ -153,7 +153,7 @@ export class OsdsInput implements OdsInputAttribute, OdsInputEvent, OdsInputMeth
 
     return (
       <Host
-        class={{ 'ods-error': this.internalError }}
```

Comment:
`class={{ 'ods-error': this.error || this.internalError }}` ?
would be nice to avoid as much as possible to allow mutable attr

Reply:
I remove the input edit for add it in the v18 PR 
https://github.com/ovh/design-system/pull/293/commits/a537288c5cc5808db9682ae7e11c2331b14e4688